### DISCUSSION TITLE
Nuxt SSR shared hosting deployment gives 404 on first load

### DISCUSSION THREAD

**Discussion by zhuxiaoyu88** [nuxt/nuxt#9502] [Questions (Nuxt 2)]
Hi everyone, I'm facing problem to deploy my project to Shared Hosting using PM2. I've succeed to deploy it, but unfortunately my site started to display "this page could not be found" on the first load, then continue to display the homepage. Anyone maybe know why and how to solve this ? Thank you

PS.
Please check out my .htaccess, nuxt.config.js, server.config.js for PM2 configuration below. I'm using `yarn build` to build my project :

**my sample web link**
http://frontend-test.timedoor.web.id/

**.htacess**
```
RewriteEngine On
RewriteRule ^ http://127.0.0.1:3001%{REQUEST_URI} [P,L]
```

**nuxt.config.js**
```
export default {
  // Global page headers (https://go.nuxtjs.dev/config-head)
  head: {
    title: 'sample-frontend',
    meta: [
      { charset: 'utf-8' },
      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
      { hid: 'description', name: 'description', content: '' },
    ],
    link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }],
  },

  // Global CSS (https://go.nuxtjs.dev/config-css)
  css: [],

  // Plugins to run before rendering page (https://go.nuxtjs.dev/config-plugins)
  plugins: [],

  // Auto import components (https://go.nuxtjs.dev/config-components)
  components: true,

  // Modules for dev and build (recommended) (https://go.nuxtjs.dev/config-modules)
  buildModules: [
    // https://go.nuxtjs.dev/eslint
    '@nuxtjs/eslint-module',
  ],

  // Modules (https://go.nuxtjs.dev/config-modules)
  modules: [
    // https://go.nuxtjs.dev/bootstrap
    'bootstrap-vue/nuxt',
    // https://go.nuxtjs.dev/axios
    '@nuxtjs/axios',
  ],

  // Axios module configuration (https://go.nuxtjs.dev/config-axios)
  axios: {},

  // Build Configuration (https://go.nuxtjs.dev/config-build)
  build: {},
}

```

**server.config.js**
```
module.exports = {
  apps : [{
    name: "sample-dev",
    script: "npm",
    args: "start",
    cwd: "mypath",
    env: {
      PORT: 3001,
      NODE_ENV: "production"
    },
    env_production: {
      PORT: 3001,
      NODE_ENV: "production"
    }
  }]
}
```

**Comment by kissu** [comment]
Working totally fine on my side. Do you have a video for this? (you can use [Loom](https://www.loom.com/) for some quick video upload)

**Reply by zhuxiaoyu88** [reply]
See the gif and the title of page "this page could not be found" 
![Nuxt load 404 first](https://user-images.githubusercontent.com/46235179/124106590-eb25ba80-da96-11eb-820e-601b2452666d.gif)

**Reply by kissu** [reply]
Can you edit your initial question with your `nuxt.config.js` file and also tell us how you do build your app?
Also, do you have a public repo for this?
